src/include/zf/zf_stack.h

```diff
@@ -167,5 +167,71 @@ enum zf_stack_feature {
 */
 ZF_LIBENTRY int zf_stack_query_feature(struct zf_stack* stack, enum zf_stack_feature feature);
 
+/*! \brief Layout for a field of statistics */
+typedef struct {
+  /** Name of statistics field */
+  char* zfsfl_name;
+  /** Offset of statistics file, in bytes */
+  int   zfsfl_offset;
+  /** Size of statistics field, in bytes */
+  int   zfsfl_size;
+} zf_stats_field_layout;
+
+/*! \brief Layout for statistics */
+typedef struct {
+  /** Size of data for statistics */
+  int                      zfsl_data_size;
+  /** Number of fields of statistics */
+  int                      zfsl_fields_num;
+  /** Array of fields of statistics */
+  zf_stats_field_layout    zfsl_fields[];
+} zf_stats_layout;
```

Comment:
I know this will be quite laborious but these structs absolutely need to have the exact same layout as their ef_vi counterparts. Can you add some build asserts to ensure the offsets and sizes of each field are the same?